src/App.vue

```diff
@@ -1,10 +1,23 @@
 <template>
-  <div id="nav">
-    <router-link to="/">Home</router-link> |
-    <router-link to="/about">About</router-link>
-  </div>
-  <router-view/>
-</template>
+  <Layout>
+    <div id="nav">
+      <router-link to="/"> {{ $t("title.register") }} </router-link> |
+      <router-link to="/about"> {{ $t("title.about") }} </router-link>
```

Comment:
We should consider centralizing the routes the use router.resolve.

Also consider refactoring to move this into a nav component